about.html

```diff
@@ -12,53 +12,83 @@
 </head>
 
 <body>
+    <header>
+      <div id="navhead">
```

Comment:
The way you styled this is good.  You can remove the list elements and move the lambda-black image into the nav and it will help reduce the amount of CSS you need to write.